src/TodoList.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
```

Comment:
Similar thing here, change this to `import { useState } from "react"`. You no longer need to import `React` in modern react.